assets/tests/components/propertiesPanel.test.tsx

```diff
@@ -156,3 +157,22 @@ describe("handleSwipe", () => {
     handleSwipe(hidePanelCB)("Right", map)
   })
 })
+
+describe("hideMeIfNoCrowdingTooltip", () => {
+  test("hides panel if no tooltip open", () => {
+    const hidePanelCB = jest.fn()
+    hideMeIfNoCrowdingTooltip(hidePanelCB)
+
+    expect(hidePanelCB).toHaveBeenCalled()
+  })
+
+  test("does not hide panel if a tooltip is open", () => {
+    const hidePanelCB = jest.fn()
+    const newDiv = document.createElement("div")
+    // @ts-ignore
+    document.getElementsByClassName = () => [newDiv]
```

Comment:
This will need to be cleaned up with `after()` to avoid polluting the global test environment. `routeLadder.test.tsx`/`routeLadder` has an example.